deps/snappy/freebsd/snappy-stubs-public.h

```diff
@@ -86,7 +86,7 @@ typedef std::string string;
   void operator=(const TypeName&)
 #endif
 
-#if !0
+#if !1
```

Comment:
Now I see why this happens. I didn't understand why you would do `#if 0` etc, but it's a generated file :)